nuxeo-dm/nuxeo-platform-webapp-core/src/main/java/org/nuxeo/ecm/webapp/action/ConversionActionBean.java

```diff
@@ -136,7 +130,7 @@ public boolean isExportableToPDF(Blob blob) {
     protected String getPDFConversionURL(Blob blob) {
         BlobManager blobManager = Framework.getService(BlobManager.class);
         try {
-            URI uri = blobManager.getAvailableConversions(blob, UsageHint.DOWNLOAD).get(PDF_MIMETYPE);
+            URI uri = blobManager.getAvailableConversions(blob, UsageHint.DOWNLOAD).get(MimetypeRegistry.PDF_MIMETYPE);
```

Comment:
You can `import static org.nuxeo.ecm.platform.mimetype.interfaces.MimetypeRegistry.PDF_MIMETYPE` for this type of constant.